stock_picking_report_cmr/views/report_cmr.xml

```diff
@@ -525,6 +551,15 @@
                                                                 Mode d'emballage
                                                                 <br />
                                                                 Method of packing
+                                                            <t t-if="o.has_packages">
+                                                                    <div
+                                                                        class="text-bigger text-dark mt-1 ml-2"
+                                                                    >
+                                                                        <span
+                                                                            t-esc="', '.join(o.move_line_ids.mapped('result_package_id.packaging_id.name'))"
```

Comment:
@gjotten this is basically package_type

Reply:
Hmm that's some ways away from strictly `package_type` on the picking - not sure about this one either, could again be something to do with their setup; @thomaspaulb

Reply:
ah I see now. stock.picking.package_type appears to be a customer specific field